### DISCUSSION TITLE
%m.view = view id?

### DISCUSSION THREAD

**Discussion by oong819** [Sketchware-Pro/Sketchware-Pro#661] [Q&A] [answered]
i have a code: `` view_id.setBackgroundColor(Color.parseColor("Color Hex code")); `` and want to turn it into a block and came up with this: `` %m.view.setBackgroundColor(Color.parseColor(%s));  `` but its dont work pls help

**Comment by hasrat-ali** [comment, marked as answer]
Replace `%m.view` with `%s` or `%1$s` in code

**Reply by oong819** [reply]
Thanks!!!
